modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/ProjectContentEventListener.java

```diff
@@ -0,0 +1,201 @@
+
+            final Project sourceProject = this.projectService.list().
+                stream().
+                filter( project -> currentProjectName.equals( project.getName() ) ).
+                findAny().
+                orElseThrow( () -> new ProjectNotFoundException( currentProjectName ) );
+
+            this.projectService.list().
+                stream().
+                filter( project -> currentProjectName.equals( project.getParent() ) ).
+                forEach( targetProject -> {
+                    final ContentId contentId = ContentId.from( nodeMap.get( "id" ) );
+
+                    final ContentEventsSyncParams.Builder paramsBuilder =
+                        ContentEventsSyncParams.create().contentId( contentId ).sourceProject( sourceProject.getName() ).targetProject(
+                            targetProject.getName() );
+
+                    switch ( type )
+                    {
+                        case "node.created":
+                            paramsBuilder.addSyncEventType( ContentSyncEventType.CREATED );
+                            break;
+                        case "node.updated":
+                        case "node.pushed":
+                            paramsBuilder.addSyncEventType( ContentSyncEventType.UPDATED );
+                            break;
+                        case "node.manualOrderUpdated":
+                            paramsBuilder.addSyncEventType( ContentSyncEventType.MANUAL_ORDER_UPDATED );
+                            break;
+                        case "node.sorted":
+                            paramsBuilder.addSyncEventType( ContentSyncEventType.SORTED );
+                            break;
+                        case "node.renamed":
+                            paramsBuilder.addSyncEventType( ContentSyncEventType.RENAMED );
+                            break;
+                        case "node.moved":
+                            paramsBuilder.addSyncEventType( ContentSyncEventType.MOVED );
+                            break;
+                        case "node.deleted":
+                            paramsBuilder.addSyncEventType( ContentSyncEventType.DELETED );
+                            break;
+                        default:
+                            LOG.debug( "Ignoring node type: {}", type );
+                            break;
+                    }
+                    final ContentEventsSyncParams params = paramsBuilder.build();
+                    if ( !params.getSyncTypes().isEmpty() )
+                    {
+                        contentSynchronizer.sync( params );
+                    }
+                } );
+
+            this.projectService.list().
+                stream().
+                filter( project -> project.getName().equals( sourceProject.getParent() ) ).
+                forEach( parentProject -> {
+
+                    final ContentId contentId = ContentId.from( nodeMap.get( "id" ) );
+
+                    switch ( type )
```

Comment:
Codacy found an issue: [Switch statements should have a default label](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6409742)